### PR TITLE
[Doc] add missing docs for the flink version 1.2.8 into branch 3.1

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/loading/Flink-connector-starrocks.md [hunk 1]
```diff
@@ -209,13 +207,13 @@ For exactly-once, the flush only happens when a checkpoint is triggered.
...
-| Metric                     | Type    | Description                                                     |
+| Name                     | Type    | Description                                                     |
 |--------------------------|---------|-----------------------------------------------------------------|
 | totalFlushBytes          | counter | successfully flushed bytes.                                     |
 | totalFlushRows           | counter | number of rows successfully flushed.                                      |
-| totalFlushSucceededTimes | counter | number of times that the data is successfully flushed.  |
-| totalFlushFailedTimes    | counter | number of times that the data fails to be flushed.                  |
-| totalFilteredRows        | counter | number of rows filtered, which is also included in totalFlushRows.    |
+| totalFlushSucceededTimes | counter | number of times that the data-batch been successfully flushed.  |
+| totalFlushFailedTimes    | counter | number of times that the flushing been failed.                  |
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:215]: number of times that the flushing has failed.

File: docs/en/loading/Flink-connector-starrocks.md [hunk 2]
```diff
@@ -485,7 +483,7 @@ This example will show how to load data only to columns `id` and `name`.
 
    - Define the DDL which only includes the columns `id` and `name`.
    - Set the option `sink.properties.partial_update` to `true` which tells the Flink connector to perform partial updates.
-   - If the Flink connector version `<=` 1.2.7, you also need to set the option `sink.properties.columns` to `id,name,__op` to tells the Flink connector which columns need to be updated. Note that you need to append the field `__op` at the end. The field `__op` indicates that the data loading is an UPSERT or DELETE operation, and its values are set by the connector automatically.
+   - If the Flink connector version <= 1.2.7, you also need to set the option `sink.properties.columns` to `id,name,__op` to tells the Flink connector which columns need to be updated. Note that you need to append the field `__op` at the end. The field `__op` indicates that the data loading is an UPSERT or DELETE operation, and its values are set by the connector automatically.
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:486]: <= 还是加上原来的符号

File: docs/en/loading/Flink-connector-starrocks.md [hunk 3]
```diff
@@ -181,7 +179,7 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
       checkpoint, instead of due to timeout (which may cause data loss).
 
   - `label_keep_max_second` and `label_keep_max_num`: StarRocks FE configurations, default values are `259200` and `1000`
-    respectively. For details, see [FE configurations](../loading/Loading_intro.md#fe-configurations). The value of `label_keep_max_second` needs to be larger than the downtime of the Flink job. Otherwise, the Flink connector can not check the state of transactions in StarRocks by using the transaction labels saved in the Flink's savepoint or checkpoint and figure out whether these transactions are committed or not, which may eventually lead to data loss.
+    respectively. For details, see [FE configurations](https://docs.starrocks.io/en-us/latest/loading/Loading_intro#fe-configurations). The value of `label_keep_max_second` needs to be larger than the downtime of the Flink job. Otherwise, the Flink connector can not check the state of transactions in StarRocks by using the transaction labels saved in the Flink's savepoint or checkpoint and figure out whether these transactions are committed or not, which may eventually lead to data loss.
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:182]: 文档引用缺 .md

File: docs/en/loading/Flink-connector-starrocks.md [hunk 4]
```diff
@@ -132,15 +129,16 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
 | DATE                              | DATE                  |
 | TIMESTAMP_WITHOUT_TIME_ZONE(N)    | DATETIME              |
 | TIMESTAMP_WITH_LOCAL_TIME_ZONE(N) | DATETIME              |
-| ARRAY&lt;T&gt;                        | ARRAY&lt;T&gt;              |
-| MAP&lt;KT,VT&gt;                        | JSON STRING           |
-| ROW&lt;arg T...&gt;                     | JSON STRING           |
+| ARRAY\<T\>                        | ARRAY\<T\>            |
+| MAP\<KT,VT\>                      | JSON STRING           |
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:133]: 应该被删除的内容才是对的吧

File: docs/en/loading/Flink-connector-starrocks.md [hunk 5]
```diff
@@ -96,22 +92,23 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
...
+| sink.properties.max_filter_ratio  | No           | 0                 | The maximum error tolerance of the Stream Load. It's the maximum percentage of data records that can be filtered out due to inadequate data quality. Valid values: `0` to `1`. Default value: `0`. See [Stream Load](../sql-reference/sql-statements/data-manipulation/STREAM_LOAD.md). for details.                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:109]: 加的 . 多余

File: docs/en/loading/Flink-connector-starrocks.md [hunk 6]
```diff
@@ -96,22 +92,23 @@ In your Maven project's `pom.xml` file, add the Flink connector as a dependency
...
+| sink.properties.strict_mode | No | false | Specifies whether to enable the strict mode for Stream Load. It affects the loading behavior when there are unqualified rows, such as inconsistent column values. Valid values: `true` and `false`. Default value: `false`. See [Stream Load](../sql-reference/sql-statements/data-manipulation/STREAM_LOAD.md). for details. |
```
Comment (evelynzhaojie) [inline docs/en/loading/Flink-connector-starrocks.md:111]: . 多余

**Comment by hellolilyliuyi** [comment]
https://github.com/StarRocks/starrocks/pull/37429/files

**Pull request closed without merging**